app/controllers/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module V0
+  class MessageDraftsController < HealthcareMessagingController
+    def create
+      response = client.post_create_message_draft(draft_params)
+      # Should we accept default Gem error handling when creating a message with invalid parameter set, or
+      # create a VA common exception?
+
+      render json: response,
+             serializer: MessageDraftSerializer,
+             meta:  {}
+    end
+
+    def update
+      params = draft_params
+      raise VA::API::Common::Exceptions::ParameterMissing, :id unless params[:id].present?
+
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageDraftSerializer,
+             meta:  {}
+    end
+
+    private
+
+    def draft_params
```

Comment:
i would create separate strong params for create vs update -- one with id, one without -- raise the parameter missing exception within there.